src/components/AppComponents/TimeoutPopup/index.tsx

```diff
@@ -63,13 +63,29 @@ export default function TimeoutPopup(props) {
       };
     }
   }, [show]);
+  function screenReaderContentDisplay() {
+    if (!isConfirmationPage) {
+      switch (isAuthorised) {
+        case true:
+          screenreadercontent = t('FOR_YOUR_SECURITY_WE_WILL_SIGN_YOU_OUT');
+          break;
+        case false:
+          screenreadercontent = t('FOR_YOUR_SECURITY_WE_WILL_DELETE_YOUR_ANSWER');
+          break;
+        default:
+      }
+    } else {
+      screenreadercontent = t('FOR_YOUR_SECURITY_WE_WILL_AUTOMATICALLY_CLOSE_IN');
+    }
+  }
```

Comment:
Instead of the extra variable, just return the translations then in the dispatch function on line 88 just call the function:
```
 payload: `${screenReaderContentDisplay()} ${t('1_MINUTE')}`
 ```